packages/apps-editor-sdk/src/types.ts

```diff
@@ -29,6 +29,7 @@ interface PayloadAppContext {
   region: region;
   timezone: string;
   appViewerToken: string;
+  appManagementToken?: string;
```

Comment:
@lbaldrickSC In the Editor SDK, I'd expect the `appManagementToken` to always be present?

Reply:
True! Will change that